### ISSUE TITLE
[Improvement] Missing bundle logger

### ISSUE THREAD

**Issue by Writtscher** [jakemmarsh/angularjs-gulp-browserify-boilerplate#182]
You have a bundle logger https://github.com/jakemmarsh/angularjs-gulp-browserify-boilerplate/blob/master/gulp/util/bundleLogger.js but you are not using it :disappointed:.

I've added these lines to my browserify file
```
...
import bundleLogger from '../util/bundleLogger;
...

function rebundle() {
    bundleLogger.start();
    const stream = bundler.bundle();
    const sourceMapLocation = global.isProd ? './' : '';

    return stream
         .on('error', handleErrors)
         .on('end', bundleLogger.end)
...
```

Works smoothly. Thanks.

**Comment by Shyam-Chen** [comment]
#173 https://github.com/jakemmarsh/angularjs-gulp-browserify-boilerplate/blob/master/gulp/tasks/browserify.js#L40

**Comment by Writtscher** [comment]
Works as well, thanks.

**Comment by Shyam-Chen** [comment]
My AngularCoffee project: https://github.com/Shyam-Chen/angular-starter-kit

**Comment by jakemmarsh** [comment]
Fix out in #183, thanks for pointing this out! Also cleaned up the code.

I don't believe @Shyam-Chen's approach would actually work, as the `rebundle` method is asynchronous. It needs to be in the `end` event as you originally had it.

**Comment by Shyam-Chen** [comment]
Yes, indeed there is no actual work

https://github.com/Shyam-Chen/angular-starter-kit/commit/35525b6d957443fa59c9bc1b14f5b5b85c7d8470
